gui/src/components/onboarding/system-config/user-settings/user-setting-local.vue

```diff
@@ -852,13 +858,109 @@ export default class CortxUserSettingLocal extends Vue {
       successDialogText: ""
     };
   }
-
+  
+  public csmUsersQueryParam: csmUserQueryParam = {sortby: '', dir: ''} as csmUserQueryParam;
   public async mounted() {
     await this.getUserData();
   }
 
+  public deleteUserActionCB(event:any, data:any){
+    this.onDeleteConfirmation(data.id);
+  }
+  public deleteActionCondition(record: any) {
+    const vueInstance: any = this;
+    if (vueInstance.$hasAccessToCsm(userPermissions.users + userPermissions.delete)) {
```

Comment:
You can directly return vueInstance.$hasAccessToCsm(userPermissions.users + userPermissions.delete)